test/all.test.e2e.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// eslint-disable-next-line node/no-extraneous-require
+const { flatMap, uniq, negate } = require('lodash/fp');
+const glob = require('glob');
+const path = require('path');
+const chalk = require('chalk');
+
+const fileExtension = '.test.e2e.js';
+const filePattern = `*${fileExtension}`;
+
+const defaultPaths = ['packages/'];
+
+const testPaths = process.env.STRAPI_TEST_PATHS;
```

Comment:
Not sure if it's relevant to add it to the doc since it's an internal variable declared and used when running the tests from the test/e2e.js script. @alexandrebodin do you have an opinion on this?

Reply:
You're right, maybe adding it in CONTRIBUTING.md ?

Reply:
I wouldn't document it as it is used internaly and should not be set manually anyway. maybe a comment in the code to explain the usage